.github/workflows/android_deploy_production_to_playstore.yml

```diff
@@ -0,0 +1,60 @@
+name: Android - Deploy Production build to Play Store
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_deploy_android:
+    name: Build & Deploy Android
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Python
```

Comment:
```suggestion
      - name: Set up Python
```

Reply:
Good point @doannimble

Reply:
Fixed in https://github.com/nimblehq/flutter-templates/pull/163/commits/6b14cdee4853dc0cc68a1eee4c2d209032b2b33b